Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/test_tools_fixtures.py

```diff
@@ -55,6 +55,9 @@ def pytest_configure(config):
     ly_test_tools._internal.pytest_plugin.build_directory = _get_build_directory(config)
     ly_test_tools._internal.pytest_plugin.output_path = _get_output_path(config)
 
+    # patch to work around TODO GHI
```

Comment:
Oops, left myself a TODO in which I intended to link to a GHI in the pytest project.